.changeset/giant-chicken-help.md

```diff
@@ -0,0 +1,5 @@
+---
+"slonik": major
```

Comment:
Note: Marked this as major. It's a bugfix but technically I think user's applications might've inadvertently relied on this buggy behaviour